config/action.d/freebsd-ipfw.conf

```diff
@@ -0,0 +1,99 @@
+# Fail2Ban configuration file
+#
+# This file was created to provide ipfw with a configuration for FreeBSD
+# as flexible as possible
+# Author:	Nilton J. Rizzo
+#
+#
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = 
+
+
+# Option:  actionstop
+# Notes.:  command executed once at the end of Fail2Ban
+# Values:  CMD
+#
+actionstop =  ipfw delete <rulenumber>
+
+
+# Option:  actioncheck
+# Notes.:  command executed once before each actionban command
+# Values:  CMD
+#
+actioncheck = 
+
+
+# Option:  actionban
+# Notes.:  command executed when banning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    See jail.conf(5) man page
+# Values:  CMD
+#
+#actionban = ipfw add <rulenumber> <blocktype> <proto> from <ip> to <localhost> dst-port <port> in via <nic> 
+actionban = ipfw add <rulenumber> <blocktype> <proto> from <ip> to <localhost> <port>
+
+
+# Option:  actionunban
+# Notes.:  command executed when unbanning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    See jail.conf(5) man page
+# Values:  CMD
+#
+actionunban = ipfw delete `ipfw list | grep -i "[^0-9]<ip>[^0-9]" | awk '{print $1;}'`
+
+[Init]
+
+# Option:  port
+# Notes.:  specifies port to monitor
+# Values:  [ NUM | STRING ]
+#
+port = ssh
+
+# Option:  localhost
+# Notes.:  the local IP address of the network interface
+# Values:  IP
+#
+#localhost = me # first nic on system
+localhost = 127.0.0.1
+
+
+# Option:  blocktype
+# Notes.:  How to block the traffic. Use a action from man 5 ipfw
+#          Common values: deny, unreach port, reset
+# Values:  STRING
+#
+blocktype = deny
+
+
+# Option:  rulenumber
+# Notes.:  Number of set of rules.  IPFW can manipulate some rules on the same number rule
+#          Common values: 10000
+# Values:  NUM
+#
+rulenumber = 10000
+
+
+
+# Option:  proto
+# Notes.:  Name of protocoll under rule
+#          Common values: tcp, udp, all, ip
+#				see ipfw man page
+# Values:  STRING
+#
+proto = ip   
+
+
+
+# Option:  nic
+# Notes.:  Name of network inteface, see /etc/rc.conf
+#          Common values: em0, re0, ....
+#				see /usr/src/sys/{ARCH}/conf/LINT for all nic names
+# Values:  STRING
+#
+nic = 
```

Comment:
Why would you want to block only one interface and not _all_ of them?
Can you provide a realistic use case where this would be needed?

So far, I cannot come up with any **realistic** scenario where this would be needed.